For now errno is still a threadvar variable.

t/tls-threadvar (when migrating to newer upstream glibc)

---
 include/errno.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/errno.h
+++ b/include/errno.h
@@ -21,7 +21,7 @@
 
 #  include <tls.h>
 
-#  if USE___THREAD
+#  if USE___THREAD && !defined(__GNU__)
 #   undef  errno
 #   ifndef NOT_IN_libc
 #    define errno __libc_errno
